README.md

```diff
@@ -84,114 +53,100 @@ To remove a dependency from your project, you can use the `poetry remove` comman
-virtual environment, simply `exit` command.
+You can now run Python and any installed packages in this shell. They will use the virtual environment. 
+To exit the virtual environment, simply use the `deactivate` command.
 
-Poetry automatically uses the virtual environment when you run commands like `poetry run`.
+Poetry automatically uses the virtual environment when you run, for example, the `poetry run` command.
 
-Remember to run these commands in the root directory of your project where the `pyproject.toml` file is located.
+Remember to run these commands in the root directory of your project, where the `pyproject.toml` file is located.
 
-Pycharm users can also use the virtual environment created by Poetry. To do this follow
-the [guides](https://www.jetbrains.com/help/pycharm/poetry.html).
+If you are a PyCharm user and want to use the virtual environment created by Poetry, follow the [configuration guides](https://www.jetbrains.com/help/pycharm/poetry.html).
 
-## Using Poe the Poet as Task Runner
+## Use Poe the Poet as Task Runner
 
 [Poe the Poet](https://poethepoet.natn.io/index.html) is a task runner that simplifies running common tasks in a Python project.
 
-To have the command availalbe as `poetry poe <command>` as seen in the following examples, you need to install Poe as a plugin to poetry:
+To have the command available as `poetry poe <command>` as seen in the following examples, you need to install poe as a plugin to Poetry:
 
 ```bash
 poetry self add 'poethepoet[poetry_plugin]'
 ```
 
-If this plugin is not installed you may have to run Poe as a script within the Poetry environment using `poetry run poe <command>`.
-
-## Usage
-
-> Explain how to use the project. You can create multiple subsections (H3). Include the instructions or provide links to
-> the related documentation.
+If this plugin is not installed, you may have to run poe as a script within the Poetry environment using `poetry run poe <command>`.
 
 ## Development
 
-This sections describes how to develop and run the kyma-companion fastapi app.
-
-### Redis Server
+### Redis Server 
 
-Before running the application, the Redis server must be provided that is used to store the conversation with a LLM.
-Therefore, REDIS_URL has to be provided as an environment variable.
+Before running the application, the Redis server must be provided. It is used to store the conversation with a large language model (LLM).
+Therefore, **REDIS_URL** must be provided as an environment variable.
 
-Here is the [documentation](https://github.tools.sap/kyma/ai-force/blob/main/docs/infrastructure/setup.md#15-redis) how to create a Redis server. For example,
-`REDIS_URL="redis://{host or ip}:6379"`
+Here is the [documentation](https://github.tools.sap/kyma/ai-force/blob/main/docs/infrastructure/setup.md#15-redis) <!--the link must be replaced when the OS documentation is available --> on how to create a Redis server. 
```

Comment:
```suggestion
For details how to create a Redis server, read [Create Redis](https://github.tools.sap/kyma/ai-force/blob/main/docs/infrastructure/setup.md#15-redis) <!--replace the link when the OS documentation is available -->. 
```

In the /infrastructure/setup.md file, I suggest using more meaningful titles, like in this case, "Create Redis" (to match the two sub-procedures of this section).